arguments_classes/open_api_language_model_arguments.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass, field
+
+
+@dataclass
+class OpenApiLanguageModelHandlerArguments:
+    open_api_model_name: str = field(
+        # default="HuggingFaceTB/SmolLM-360M-Instruct",
+        default="deepseek-chat",
+        metadata={
+            "help": "The pretrained language model to use. Default is 'microsoft/Phi-3-mini-4k-instruct'."
+        },
+    )
+    open_api_user_role: str = field(
+        default="user",
+        metadata={
+            "help": "Role assigned to the user in the chat context. Default is 'user'."
+        },
+    )
+    open_api_init_chat_role: str = field(
+        default="system",
+        metadata={
+            "help": "Initial role for setting up the chat context. Default is 'system'."
+        },
+    )
+    open_api_init_chat_prompt: str = field(
+        # default="You are a helpful and friendly AI assistant. You are polite, respectful, and aim to provide concise responses of less than 20 words.",
+        default="你是一位乐于助人且友好的 AI 助手。您彬彬有礼、尊重他人.",
```

Comment:
Let's stick with a default initial chat prompt in english. 

```suggestion
        default="You are a helpful and friendly AI assistant. You are polite, respectful, and aim to provide concise responses of less than 20 words.",
```